diagnostic.md

```diff
@@ -36,13 +41,26 @@ Now, we want to get a single donut from the server. Write the `curl` request
 you'd use to retrieve a single donut, using whatever ID you'd like.
 
 ```sh
-# your answer here
+curl --include --request GET 'http://www.example.com/donuts/10'
 ```
 
 Write an AJAX request to retrieve a single donut from the server.
 
 ```js
-let getDonut = /* your answer here */;
+let getDonut = = function (event){
+  event.preventDefault();
+  let donutId = $(event.target).find('[name="donut[id]"]').val();
+
+  if (donutId.length === 0) {
+    libraryApi.index()
+      .done(ui.logResponseBody)
+      .fail(ui.logRequestError);
+  } else {
+    libraryApi.show(event.target)
+      .done(ui.logResponseBody)
+      .fail(ui.logRequestError);
+  }
+};
 ```
```

Comment:
1/2. I see what you wanted to do here but this is not an ajax request, the libraryApu.index() function that this calls would be the ajax request